simulation/test_torrent_status.cpp

```diff
@@ -41,15 +41,38 @@ using namespace sim;
 namespace lt = libtorrent;
 
 // this is a test for torrent_status time counters are correct
+// it is a long running test, we did already try to improve it:
+// - less slow peers vs many fast peers
+// - slow peers with smal torrent vs fast peers with large torrent
+// - less peers with large torrents vs more peers with smal torrent
+// - connection limit vs parallel requests
+// as conlusion the following combination take the least amount of
+// execution time per tick:
+// - many slow peers with small torrent and connection limit
 TORRENT_TEST(status_timers)
 {
 	lt::time_point startseed_time;
+	lt::time_point last_active_time;
 	lt::torrent_handle handle;
 	int last_tick;
+	// at least 11s intervall because after resume it takes 11s untill we unchuck
+	// new pieces at this slow upload rate
+	int ticks_check_intervall = 20;
+	seconds active_time = seconds(0);
+	bool tick_is_in_active_range = false;
 
-	setup_swarm(2, swarm_test::upload
+	setup_swarm(10, swarm_test::upload_slow
```

Comment:
I did head 1000 before, only one node would use 70 ticks with this torrent size and rate limit, 70 ticks are still a little bit tight for testing so much.

But there is no much difference beteween 2 and 10, only 3-5s less execution time. Lowering upload rate limit would make it worse becaus you get more ticks without upload activity. Larger torrents should help. I could change it to 2 with a 5 times larger torrent.
Maybe it would be nice to have 10 nodes for the torrent peer timestamp test. But maybe that is to complicated to test and I change it to 2 nodes with 5 times larger torrent anyway.